### DISCUSSION TITLE
is the Accounts icon on the bottom left of sidebar supposed to work?

### DISCUSSION THREAD

**Discussion by xuchen** [coder/code-server#2568] [Q&A]
![image](https://user-images.githubusercontent.com/543547/104277591-f8ed7200-545b-11eb-8d5d-f0ae046e9e11.png)

When I click on it, it always shows "Loading...". I'm using this on a remote container via https access. 

If it's not supposed to work (I think settings sync is recommended via a separate extension), maybe hide this icon by default?

**Comment by code-asher** [comment]
Yeah, removing it sounds like the right play here. I haven't had a
chance to see yet if we can get it working and I'm not sure when I'll be
able to poke into it.

**Reply by ghost** [reply]
It works for me after I sign into github when prompted by the git extension. That happens when you select publish to github from the source control panel in the activity bar. Maybe make it visible only if user signed into github?

**Reply by code-asher** [reply]
It does?? That's awesome. Maybe we can add a "Sign in with GitHub" button there.

**Reply by ghost** [reply]
That would be much easier than it only happening when you need to publish to github. Also would make it easier to find.

**Reply by jsjoeio** [reply]
Great idea! Opened an issue here: https://github.com/cdr/code-server/issues/2913

**Reply by nooobcoder** [reply]
Do not remove it, @code-asher , maybe there in near future there would be something like **coder id** to log in to it and sync settings to code-server without any third party extensions just like vscode does.

How is the idea?

**Reply by jsjoeio** [reply]
@nooobcoder Feel free to leave any comments/suggestsions on the issue we opened #2913
